circle.yml

```diff
@@ -0,0 +1,14 @@
+machine:
+  services:
+    - docker
+
+dependencies:
+  override:
+    - docker info
+    - docker pull lucasgandel/slicerexecutionmodel
```

Comment:
I created a slicer organization and a repository named slicerexecutionmodel. 

User `lucasgandel` should have admin access to that repo.

See https://hub.docker.com/r/slicer/slicerexecutionmodel/

Cc: @thewtex @LucasGandel